### ISSUE TITLE
Stacking Graphical Issue

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#13728]
### Quick summary

The game seems to have graphical issues

### Details

When testing this game I ran across an issue where there's a ghosting issue which covers most of the screen.
 
Rpcs3:
![NPHB00312_screenshot_2023_04_22_14_36_38](https://user-images.githubusercontent.com/71127352/233790340-1274a294-36ed-436f-bd49-3a76ce5b172d.png)

Turning Cpu blit and WCB seem to fix the ghosting issue but seem to be really burry compared to hardware.
Rpcs3
![NPHB00312_screenshot_2023_04_22_14_48_12](https://user-images.githubusercontent.com/71127352/233790346-f9edb435-a039-4466-85a9-e4aa435258dd.png)

Ps3
https://www.youtube.com/watch?v=Dpm6zR3S0eU

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11301554/RPCS3.log.gz)

### Attach capture files for visual issues

https://drive.google.com/file/d/1g2R0_IkymOu3vkCeH3vKqin_FcYyrSMF/view?usp=share_link  (Default Setting)
https://drive.google.com/file/d/16IfYdGt2569oNlkVLtORRfq8S4jpDcAO/view?usp=share_link (Cpu blit + WCB )

### System configuration

RPCS3 v0.0.27-14905-bb8e43f1 Alpha | master | FW 4.88 | 5.15.106-1-MANJARO
AMD Ryzen 5 2600 | 12 Threads | 15.95 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (530.41)

**Comment by Triticum0** [comment]
Restested with strict setup + Force Hardware MSAA Resolve and make no diffrence

**Comment by digant73** [comment]
is that issue still present?

**Comment by Triticum0** [comment]
Yes it but it seem to be a regression when using Cpu blit look much worse.

**Comment by alexconsole04** [comment]
This blur also happens in Brutal Legend when both settings are enabled. While not required, it's still something interesting to point out since these two games share the same engine (Buddha.) Costume Quest 1 from the looks of it does not have this blur problem despite being the same engine. This doesn't add to the original issue, just something I wanted to point out.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/5f8ed454-c3c9-4f05-9477-e2e5dfcaf982" />

**Comment by ShadowH69** [comment]
This can be closed , confirmed fixed after #19013 

<img width="1919" height="997" alt="Image" src="https://github.com/user-attachments/assets/9acf816e-ddfb-46f9-b10d-dbfc22c6c4c5" />
